packages/jsapi-utils/src/MessageUtils.ts

```diff
@@ -127,7 +124,7 @@ export async function requestParentResponse<T>(
       }
       window.clearTimeout(timeoutId);
       window.removeEventListener('message', listener);
-      resolve(data.payload);
+      resolve(data.payload as T);
```

Comment:
I think I'd rather this util return `Promise<unknown>` since it doesn't do any validation. Then it's up to consumption to either validate it or cast it if they don't care about the validation really

Just makes the util more clear